pkg/controller/config/config_controller.go

```diff
@@ -828,3 +837,16 @@ func sourceBasedOnRecursion(path string) mf.Source {
 	}
 	return mf.Path(path)
 }
+
+func createAddonsPipelines(mgr manager.Manager) (mf.Manifest, error) {
+	pipelineTemplate, err := mf.NewManifest("deploy/resources/templates/pipelinetemplate.yaml")
```

Comment:
move this path to `pkg/flag.go` and make you of `path.Join()`

Reply:
I used the path.Join() earlier but then directly used complete path, do we have any specific reason to use path.Join().

Reply:
yes, people might run tests from different platforms, where platforms are `/` or `\`